app/helpers/operatingsystems_helper.rb

```diff
@@ -58,7 +58,10 @@ def icon(record, opts = {})
            when /Windows/i
              "stub/steelblue-w"
            else
-             return os_image_tag(size, "stub/black-x", opts) if record.family.blank?
+             if record.family.blank?
+               return image_path("icons#{size}/stub/black-x.png") if opts[:path]
+               return os_image_tag(size, "stub/black-x", opts)
+             end
              record.family.downcase
```

Comment:
How about
```ruby
if record.family.blank?
  'stub/black-x'
else
  record.family.downcase
end
```

I think that's essentially the same, but keeps the structure of the method more more the same.

Reply:
Nice improvement! :+1: